maturity-model.md

```diff
@@ -0,0 +1,175 @@
+* CB-3:	Members of the organization collaborate both internally and externally in ways that benefit all involved. Teams routinely discuss, revisit and debate the outcomes of their collaborative efforts, and share their learnings outside the organization, and make these outcomes externally available by default.
+
+### Community
+
+**Sharing Policies**
+
+* SP-0: No sharing culture nor written policies.
+* SP-1:	Some members of the organization unite to define values and principles, but are not clearly supported when they do.
+* SP-2:	Members of the organization collectively document shared visions and agreements like mission statements and codes of conduct, make them easily accessible, and reference them often. Onboarding materials and orientation rituals provide adequate context for helping new members understand how the organization will benefit from their contributions.
+* SP-3:	Shared values and principles inform decision-making, conflict resolution, and assessment processes among members of the organization, who reference these values and principles consistently in both verbal and written formats.
+
+**Feel part of the Organization**
+
+* PA-0:	Low engagement, no collaboration and people do not feel comfortable sharing with others.
+* PA-1:	Members of the organization feel comfortable sharing their thoughts and opinions without fear of retribution, but only in familiar domains. People understand that the best ideas win, and leadership responsibilities accrue to people with histories of contribution and commitment.
+* PA-2:	Members of the organization feel comfortable sharing their thoughts and opinions without fear of retribution. Leaders demonstrate dedication to the organization's shared values.
+* PA-3:	The organization is proactive in telling members that it benefits from their contributions; as such, members demonstrate shared consciousness and empowered execution, and feel a sense of responsibility to the community. Leaders understand that they grow by helping others grow, and they mentor junior members of the organization.
+
+### Governance
+
+**Rewards**
+
+* RW-0:	No rewards.
+* RW-1:	Leaders are encouraged to reward exceptional collaborations, but there are no policies or processes established.
+* RW-2:	Standard processes are established to reward collaborations outside the developers' teams. Team leaders or boards decide who has to be rewarded.
+* RW-3:	Rewards are not only proposed by the organization, but the communities are able to define their more valuable rewards. The community is responsible to decide who has to be rewarded.
+
+**Monitoring Policies**
+
+* MP-0:	No existing monitoring policies.
+* MP-1:	Some people start using metrics for their own needs at their own hierarchical level.
+* MP-2:	Existing process at the organization.
+* MP-3:	Clear rules and processes in the use of metrics with specific training and monitoring in place.
+
+**Support and Maintenance**
```

Comment:
Again this sounds more like a point that is relevant for pretty much any tech department and not InnerSource specific.

To make it InnerSource specific we could talk about adoption of the 30-day warranty and service vs. library patterns.

Reply:
That's a good idea: link to existing patterns or to the creation of new patterns to open certain discussions.

I think it makes sense to have this across the model.